app/docs/0.10.x/proxy.md

```diff
@@ -0,0 +1,759 @@
+the `uris` field. Example:
+
+```json
+{
+    "name": "my-api",
+    "upstream_url": "http://my-api.com",
+    "uris": ["/service", "/hello/world"]
+}
+```
+
+The following requests would match the configured API:
+
+```http
+GET /service HTTP/1.1
+Host: my-api.com
+```
+
+```http
+GET /service/resource?param=value HTTP/1.1
+Host: my-api.com
+```
+
+```http
+GET /hello/world/resource HTTP/1.1
+Host: anything.com
+```
+
+For each of those requests, Kong detects that their URI is prefixed with one of
+the API's `uris` values. By default, Kong would then forward the request 
+upstream with the untouched, **same URI**.
+
+When proxying with URIs prefixes, **the longest URIs get evaluated first**. This
+allow you to define two APIs with two URIs: `/service` and `/service/resource`,
+and ensure that the former does not "shadow" the later.
+
+[Back to TOC](#table-of-contents)
+
+##### The `strip_uri` property
+
+Sometimes, you might want to specify a URI prefix to match an API, but not
+include it in the upstream request. You can use the `strip_uri` boolean 
+property for that:
+
+```json
+{
+    "name": "my-api",
+    "upstream_url": "http://my-api.com",
+    "uris": ["/service"],
+    "strip_uri": true
+}
+```
+
+Enabling this flag instructs Kong that when proxying this API, it should **not**
+include the matching URI prefix in the upstream request's URI. For example, the
+following client's request:
+
+```http
+GET /service/path/to/resource HTTP/1.1
+Host:
+```
+
+Will make Kong send the following request to your upstream service:
+
+```http
+GET /path/to/resource HTTP/1.1
+Host: my-api.com
+```
+
+This property can come very handy at times.
+
+[Back to TOC](#table-of-contents)
+
+#### Request HTTP method
+
+Starting with Kong 0.10, client requests can also be routed depending on their
+HTTP method by specifying the `methods` field. By default, Kong will route a 
+request to an API regardless of its HTTP method. But when this field is set, 
+only requests with the specified HTTP methods will be matched.
+
+This field also accepts multiple values. Here is an example of an API allowing
+routing via a simple HTTP method:
+
+```json
+{
+    "name": "api-1",
+    "upstream_url": "http://my-api.com",
+    "methods": ["GET", "HEAD"]
+}
+```
+
+Such an API would be matched with the following requests:
+
+```http
+GET / HTTP/1.1
+Host:
+```
+
+```http
+HEAD /resource HTTP/1.1
+Host:
+```
+
+But would not match a `POST` or a `DELETE` request. This allows for much more 
+granularity when configuring APIs and Plugins. For example, one could imagine 
+two APIs pointing to the same upstream service: one allowing for `GET` requests
+and unprotected, and a second one filtering out `POST` requests and applying 
+authentication or rate limiting plugins on such requests.
+
+[Back to TOC](#table-of-contents)
+
+### Routing priorities
+
+It is now clear that an API may define matching rules based on its `hosts`, 
```

Comment:
"It is now clear" sentences seem unnecessary. If it's necessary to restate these conditions, let's reword this to not use a condescending tone. Otherwise, we can scrap these two sentences.